Project.toml

```diff
@@ -4,10 +4,13 @@ authors = ["Guillaume Fraux <guillaume.fraux@epfl.ch>"]
 version = "0.10.3"
 
 [deps]
+AtomsBase = "a963bdd2-2df7-4f54-a1ee-49d51e6be12a"
 BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
 Chemfiles_jll = "78a364fa-1a3c-552a-b4bb-8fa0f9c1fcca"
 DocStringExtensions = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+PeriodicTable = "7b2266bf-644c-5ea3-82d8-af4bbd25a884"
+Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
```

Comment:
Why are these dependencies also required? Are them transitive dependencies coming from AtomsBase?

Reply:
Sort of, both PeriodicTable and Unitful are used by AtomsBase internally. However both of them are being used by AtomsBase.jl in Chemfiles as well because the AtomsBase interface requires most functions to return Unitful values. The function element(atom::Atom) is supposed to return a PeriodicTable.Element object.